apps/condo/domains/ticket/access/Ticket.js

```diff
@@ -111,6 +115,30 @@ async function canManageTickets (args) {
             return ticket.client === user.id
         }
     } else {
+        if (isBulkRequest && operation === 'create') return false
```

Comment:
you already check it in line 73

Reply:
Yes, that's right. I thought about leaving the check for better understanding and easier scaling of the code in the future. But if it's unnecessary, I'll remove it. Let me know

Reply:
I think it's better to remove it and leave in one place